### PR TITLE
Get Subscription form a Plan

### LINKED ISSUE #97: Get Subscriptions from a Plan
New route:
`GET /plans/plan_id/subscriptions`

### PR DESCRIPTION
**Pull request by GabrielBlondeau-Kirby**
fix #97  

---

New route 
`GET /plans/plan_id/subscriptions`

### CODE REVIEW

File: source/includes/_plans.md [hunk 1]
```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
...
+Authorization: Bearer <jwt>
+X-Kvass-Api-Key: <kvass-api-key>
+Host: api.kvass.ai
+```
+
+``` http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
```
Comment (KristopherJordan) [inline source/includes/_plans.md]: The response are dicts in a list, isnt it?

File: source/includes/_plans.md [hunk 2]
```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
...
+
+Argument | Type | Description
+-------- | ---- | -----
+**plan_id** | `string` | ID of the queried plan.
+size | `integer` | Number of items to retrieve.
+page | `integer` | Which page to retrieve. _default page size is 50_
+sorting | `string`| Field used for sorting results. <br> If missing, the default is `-created`.
+include_deleted | `boolean` | If `true`, deleted products are also listed.
+status | `string` | Return subscriptions with a specific [`Subscription`](#subscriptions) `status`.
+method | `string` | Return subscriptions with a specific [`Payment Method`](#payment_methods) `method`.
```
Comment (KristopherJordan) [inline source/includes/_plans.md]: payment method?

File: source/includes/_plans.md [hunk 3]
```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
...
+}
+
+```
+
+Argument | Type | Description
+-------- | ---- | -----
+**plan_id** | `string` | ID of the queried plan.
+size | `integer` | Number of items to retrieve.
+page | `integer` | Which page to retrieve. _default page size is 50_
+sorting | `string`| Field used for sorting results. <br> If missing, the default is `-created`.
```
Comment (KristopherJordan) [inline source/includes/_plans.md]: isn't it `sort`?

File: source/includes/_plans.md [hunk 4]
```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
 total_amount |`number`| The total cost of the subscription payment. If set, the total_amount is not based on the sum of products.
 recurring_days | `array` | When updating the recurring days you can return both of following format:
  - `{"day": 0, "starting_day": {"$date": 1496410494652}, "next_day": {"$date": 1496410494652}}` - it means this recurring day already exists
- - `{"day": 2, "hour": 10, "minute": 15}`
\ No newline at end of file
+ - `{"day": 2, "hour": 10, "minute": 15}`
+ 
+ 
+ ### Get Subscription used by a Plan
```
Comment (KristopherJordan) [inline source/includes/_plans.md]: `Get subscriptions used by a Plan`

File: source/includes/_plans.md [hunk 5]
```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
...
+
+You can know if your plan is used in [`Subscription`](#subscriptions) by using the following request which will return you a list of [`Subscription`](#subscriptions).
```
Comment (KristopherJordan) [inline source/includes/_plans.md]: This request will return an array of all subscriptions used by a plan.

File: source/includes/_plans.md [hunk 6]
```diff
@@ -508,4 +506,65 @@ discount |`number`| Between 0.0 an 1.0, where 1.0 is 100% discount on the `total
...
+Allowing you to get all [`Subscription`](#subscriptions) or some by using filter like `status` or `method` as a parameters.
```
Comment (KristopherJordan) [inline source/includes/_plans.md]: We also allow you to use query parameters. Please take a look below for a list of accepted query parameters.

File: source/includes/_plans.md (hunk 2 above)
Comment (GabrielBlondeau-Kirby) [reply source/includes/_plans.md]: :thinking: hmmm Why did I write that?!

**Pull request merged**